recipes/fiftyone/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "fiftyone" %}
+{% set version = "0.24.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiftyone-{{ version }}.tar.gz
+  sha256: 451b4f24496ee610bd76d0d344722a4e71c7636c71d80e9e11813bf330bfe44e
+  patches:
+    - 0000-relax-deps.patch
+    - 0000-dropin-fiftyone-db.patch
+    - 0000-allow-newer-strawberry.patch
```

Comment:
Are these upstream? If so, can you add a comment on top of each line with the URL for the PR? If not a quick explanation on why they are needed?

Reply:
This strawberry stuff I got merged already. IDK if it is already available in a release.

Relax deps is mostly because they ship mongodb as fiftyone-db and here I wanted to patch it to use the mongodb conda package.

How can I proceed in this case?

Reply:
Strawberry PR https://github.com/voxel51/fiftyone/pull/4697

Reply:
Please add the comments in the recipe. They are so we remember why we patched.

Reply:
Done